pom.xml

```diff
@@ -5,6 +5,14 @@
     <groupId>com.xmslyz.luhn</groupId>
     <artifactId>LuhnAlgorithm</artifactId>
     <version>1.0-SNAPSHOT</version>
+    <dependencies>
+        <dependency>
+            <groupId>org.junit.jupiter</groupId>
+            <artifactId>junit-jupiter</artifactId>
+            <version>RELEASE</version>
+            <scope>compile</scope>
+        </dependency>
+    </dependencies>
```

Comment:
PR powinien być spójny tematycznie, testami możemy się zająć w kolejnym. Poza tym powinno to wyglądać tak:
```
    <dependencies>
        <dependency>
            <groupId>org.junit.jupiter</groupId>
            <artifactId>junit-jupiter</artifactId>
            <version>5.9.1</version>
            <scope>test</scope>
        </dependency>
    </dependencies>
```
wersję lepiej podawać explicite, żeby maven nie ściągnął przypadkowej (bo nagle i niespodziewanie projekt może przestać budować się!). A junit to zależność tylko do testów, nie chcesz od niego zależeć w kodzie "produkcyjnym"